// [[Reason: Syntactically invalid. Duplicate property]] ////

//// [tests/cases/conformance/expressions/typeAssertions/duplicatePropertiesInTypeAssertions02.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -2,8 +2,9 @@
 
 //// [duplicatePropertiesInTypeAssertions02.d.ts]
 declare let x: {
     a: number;
+    a: number;
 };
 //# sourceMappingURL=duplicatePropertiesInTypeAssertions02.d.ts.map
 /// [Errors] ////
 
